src/About.scss

```diff
@@ -77,7 +81,7 @@
       }
 
       .name {
-        color: $color2;
+        color: $orange;
```

Comment:
@samjianga consider using generic variable names such as primary-color, secondary-color, etc. That way if we change the colors later that variable name still makes sense.